provider/resources.go

```diff
@@ -97,105 +101,109 @@ func Provider() tfbridge.ProviderInfo {
+			"akamai_appsec_rate_policy":                     {Tok: makeResource(mainMod, "AppSecRatePolicy")},
+			"akamai_appsec_rate_policy_action":              {Tok: makeResource(mainMod, "AppSecRatePolicyAction")},
+			"akamai_appsec_rate_protection":                 {Tok: makeResource(mainMod, "AppSecRateProtection")},
+			"akamai_appsec_reputation_profile":              {Tok: makeResource(mainMod, "AppSecReputationProfile")},
+			"akamai_appsec_reputation_profile_action":       {Tok: makeResource(mainMod, "AppSecReputationProfileAction")},
+			"akamai_appsec_reputation_profile_analysis":     {Tok: makeResource(mainMod, "AppSecReputationProfileAnalysis")},
+			"akamai_appsec_reputation_protection":           {Tok: makeResource(mainMod, "AppSecReputationProtection")},
+			"akamai_appsec_rule_upgrade":                    {Tok: makeResource(mainMod, "AppSecRuleUpgrade")},
+			"akamai_appsec_security_policy":                 {Tok: makeResource(mainMod, "AppSecSecurityPolicy")},
+			"akamai_appsec_security_policy_rename":          {Tok: makeResource(mainMod, "AppSecSecurityPolicyRename")},
+			"akamai_appsec_siem_settings":                   {Tok: makeResource(mainMod, "AppSecSiemSettings")},
+			"akamai_appsec_slow_post":                       {Tok: makeResource(mainMod, "AppSecSlowPost")},
+			"akamai_appsec_slowpost_protection":             {Tok: makeResource(mainMod, "AppSecSlowPostProtection")},
+			"akamai_appsec_version_notes":                   {Tok: makeResource(mainMod, "AppSecVersionNodes")},
+			"akamai_appsec_waf_mode":                        {Tok: makeResource(mainMod, "AppSecWafMode")},
+			"akamai_appsec_waf_protection":                  {Tok: makeResource(mainMod, "AppSecWafProtection")},
+			"akamai_appsec_advanced_settings_pragma_header": {Tok: makeResource(mainMod, "AppSecAdvancedSettingsPragmaHeader")},
+			"akamai_appsec_api_constraints_protection":      {Tok: makeResource(mainMod, "AppSecApiConstraintsProtection")},
+			"akamai_appsec_attack_group":                    {Tok: makeResource(mainMod, "AppSecAttackGroup")},
+			"akamai_appsec_eval_rule":                       {Tok: makeResource(mainMod, "AppSecEvalRule")},
+			"akamai_appsec_ip_geo_protection":               {Tok: makeResource(mainMod, "AppSecIPGeoProtection")},
+			"akamai_appsec_rule":                            {Tok: makeResource(mainMod, "AppSecRule")},
+
+			"akamai_cps_dv_enrollment": {Tok: makeResource(mainMod, "CpsDvEnrollment")},
+			"akamai_cps_dv_validation": {Tok: makeResource(mainMod, "CpsDvValidation")},
+
+			"akamai_networklist_activations":  {Tok: makeResource(mainMod, "NetworkListActivations")},
+			"akamai_networklist_description":  {Tok: makeResource(mainMod, "NetworkListDescription")},
+			"akamai_networklist_network_list": {Tok: makeResource(mainMod, "NetworkList")},
+			"akamai_networklist_subscription": {Tok: makeResource(mainMod, "Subscription")},
```

Comment:
Would it make sense to call it `NetworkListSubscription` as others above?

Reply:
Great catch! Thank you - you are correct!